migration/idempotent/008-tracing-functions.sql

```diff
@@ -355,7 +355,7 @@ BEGIN
             _value
```

Comment:
line 343-347 and 362-367 should be changed to include where clauses on the hash `pg_catalog.md5(value::TEXT) = pg_catalog.md5(_value::TEXT)` so that the new unique index `tag_key_value_id_key_id_key` is fully used, right (or is Postgres smart enough to figure this out)

Reply:
Yeah, as the description states: it's not smart enough and there might be hidden performance problems. The ones you pointed out here, equality operators, possibly more. 

First I want to run e2e and make sure nothing is broken, then I'll do an optimization sweep.